### DISCUSSION TITLE
storage/hwameistor/intro/drbd/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1679] [Announcements]
# storage/hwameistor/intro/drbd/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/storage/hwameistor/intro/drbd/

**Comment by liyuerich** [comment]
DCE5 global集群 画眉高可用 如何启用，能介绍一下吗？
